<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLightDialog</class>
 <widget class="QDialog" name="AddLightDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Light</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralWidget
{
	background-color: #303030;
}

QDialog
{
	background-color: #303030;
}

QSplitter
{
	background-color: #303030;
}

QMenuBar
{
	background-color: #303030;
	color: #CCCCCC;
}

QMenu
{
	background-color: #303030;
	color: #CCCCCC;
}

QStatusBar
{
	background-color: #303030;
}

QStatusBar::item
{
	color: #CCCCCC;
	border-radius: 1px;
}

QToolBar
{
	background-color: #505050;
}

QMenuBar::item
{
	background-color: #303030;
}

QMenuBar::item::selected
{
	background-color: #909090;
}

QMenu::item::selected
{
	background-color: #909090;
}

QPushButton
{
	color: #CCCCCC;
	background-color: #606060;
	border-bottom-left-radius: 5px;
	padding: 5px;
}

QPushButton:hover
{
	background-color: #707070;
}

QPushButton:disabled
{
	color: #606060;
	background-color: #202020;
}

QFrame#frame
{
	background-color: #303030;
	border-bottom-left-radius: 5px;
}

QFrame#frame_2
{
	background-color: #303030;
	border-bottom-left-radius: 5px;
}

QFrame#frame_3
{
	background-color: #303030;
	border-bottom-left-radius: 5px;
}

QLabel
{
	color: #CCCCCC;
}

QListWidget
{
	background-color: #606060;
	color: #CCCCCC;
	border-radius: 1px;
}

QListWidget::item:selected
{
	background-color: #808080;
	color: #CCCCCC;
	border-radius: 1px;
}

QTreeWidget
{
	background-color: #606060;
	color: #CCCCCC;
	border-radius: 1px;
}

QLineEdit
{
	background-color: #606060;
	color: #CCCCCC;
	border-radius: 1px;
}

QGroupBox
{
	color: #CCCCCC;
}

QComboBox
{
  background-color: #606060;
  border-radius: 2px;

  selection-color: #AAAAAA;
  selection-background-color: #606060;
  show-decoration-selected: 0;
}

QComboBox:on
{
  padding-top: -10px;
  padding-left: 2px;
}

QComboBox::drop-down
{
   subcontrol-origin: padding;
   subcontrol-position: top right;
   width: 7px;

   border-left-width: 0px;
   border-left-color: #000000;
   border-left-style: solid;
   border-top-right-radius: 3px;
   border-bottom-right-radius: 3px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Radius: </string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="radiusInput"/>
   </item>
   <item row="2" column="2">
    <widget class="Color_Wheel" name="colorWheel" native="true"/>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="nameInput"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name: </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Color_Wheel</class>
   <extends>QWidget</extends>
   <header>color_wheel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddLightDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddLightDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
